service/history/shard/controller_test.go

```diff
@@ -431,9 +423,8 @@ func (s *controllerSuite) TestHistoryEngineClosed() {
 		go func() {
 			for attempt := 0; attempt < 10; attempt++ {
 				for shardID := int32(1); shardID <= numShards; shardID++ {
-					engine, err := s.shardController.GetShardByID(ctx, shardID)
+					_, _, err := s.shardController.GetShardByID(ctx, shardID)
```

Comment:
why remove this engine not nil assert?

Reply:
It didn't seem worth it but I can add it back